cuebot/src/main/java/com/imageworks/spcue/service/JobSpec.java

```diff
@@ -280,7 +277,7 @@ private BuildableJob handleJobTag(Element jobTag) {
         job.user = user;
         job.uid = uid;
         job.email = email;
-        job.os = DEFAULT_OS;
+        job.os = null; // default to no OS specified
```

Comment:
@bcipriano suggested we remove this altogether which I'm supportive of.  It would be good to get other peoples thoughts.

Reply:
I'm also i support of this. There's many cases where you don't care what OS the job runs on in a multi-platform environment.

Reply:
@jrray Flagging this change for SPI.